### PR TITLE
docs: use native uuid example

### PR DESCRIPTION
**Pull request by twlite**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,13 +299,11 @@ Or if you want to use UUID primary keys:
...
 
 ```typescript
-import { v4 } from 'uuid';
-
 @Entity()
 export class UuidBook {
 
   @PrimaryKey()
-  uuid = v4();
+  uuid = crypto.randomUUID();
```
Comment (B4nan) [inline README.md:306]: don't you need to import crypto first?

Comment (twlite) [reply README.md:306]: It is a global object that works even in browsers

Comment (B4nan) [reply README.md:306]: maybe nowadays, we support node 18, and from what i see this was made global in node 19, so lets add the explicit import please.

File: README.md [hunk 2]
```diff
@@ -299,13 +299,13 @@ Or if you want to use UUID primary keys:
 **`./entities/UuidBook.ts`**
 
 ```typescript
-import { v4 } from 'uuid';
+import { randomUUID } from 'crypto';
```
Comment (B4nan) [inline README.md:302]:
```suggestion
import { randomUUID } from 'node:crypto';
```

**Review by B4nan — APPROVED** [review]
all right, thanks! 

if you care, there will be many places like this in the actual docs, so PR welcome for those

**Pull request merged**